packages/oui-numeric/src/numeric.controller.js

```diff
@@ -0,0 +1,117 @@
+import _ from 'lodash'
+
+// By design, value is restricted to [0, 99999] interval
+const MIN_VALUE = 0
+const MAX_VALUE = 99999
+
+export default class {
+  constructor ($attrs, $element, $log) {
+    'ngInject'
+
+    this.$attrs = $attrs
+    this.$element = $element
+    this.$log = $log
+  }
+
+  $postLink () {
+    this.$element.removeAttr('id')
+    this.$element.removeAttr('name')
+  }
+
+  $onInit () {
+    if (!this.id) {
+      this.$log.warn(`Missing required attribute id`)
+    }
+
+    if (!this.text) {
+      this.$log.warn(`Missing required attribute text`)
+    }
+
+    if (!angular.isNumber(this.min)) {
+      if (angular.isDefined(this.min)) {
+        this.$log.warn(`Invalid attribute min, expected number got '${this.min}'`)
+      }
+      this.min = MIN_VALUE
+    }
+
+    if (!angular.isNumber(this.max)) {
```

Comment:
Maybe we could export that in a method like `setNumberAttr (attrName, defaultValue)` ?

We'll get:
``` 
this.setNumberAttr('min', MIN_VALUE)
this.setNumberAttr('max', MAX_VALUE)
```

Reply:
I really don't this idea. First, i'm not setting an attribute here, i'm checking if this.max is a number and in case it's not i log the issue and defaults its value to MAX_VALUE.
What would you want, some kind of "checkIfNumberOrDefaultsTo()" function ? :)

Such simple logic doesn't need to be refactored in a function. IMHO the function will be hiding the control flow leading to less code readability. Feels free to comment if you disagree ;)

Reply:
It was not just for the checks, but also for the warning message. Same in the 3 cases.
It's not like we do a method just for one `if` :)